CRM/Core/BAO/WordReplacement.php

```diff
@@ -66,11 +66,11 @@ public static function getWordReplacement($reset = NULL) {
   }
 
   /**
-   * Save the values of a WordReplacement.
+   * Deprecated update function.
    *
+   * @deprecated
    * @param array $params
    * @param int $id
-   *
    * @return array
    */
   public static function edit(&$params, &$id) {
```

Comment:
I don' t think this function is ever called - we should go that step further & add a deprecation notice 

We could go that step further & add a deprecation notice to the v3 function if it were to call something else itself - I think we could add in something like `_civicrm_api3_basic_create` that would call apiv4 & return those results & start deprecating create functions where we can really migrate off them

Reply:
That's a longer-term goal I think.